DataBaseTutorial/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -0,0 +1,86 @@
+﻿// <auto-generated />
+using System;
+using DataBaseTutorial;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace DataBaseTutorial.Migrations
+{
+    [DbContext(typeof(DatabaseContext))]
+    partial class DatabaseContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "3.1.4");
+
+            modelBuilder.Entity("DataBaseTutorial.Product", b =>
+                {
+                    b.Property<int>("ProductId")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
```

Comment:
No need to specify type explicitly. EF is capable of mapping CLR types to SQL types